rasa_nlu/tokenizers/__init__.py

```diff
@@ -0,0 +1,19 @@
+def tokenizer_from_name(name, language):
+    from rasa_nlu.tokenizers.whitespace_tokenizer import WhitespaceTokenizer
+    from rasa_nlu.tokenizers.mitie_tokenizer import MitieTokenizer
+    from rasa_nlu.tokenizers.spacy_tokenizer import SpacyTokenizer
+
+    if name == MitieTokenizer.name:
+        return MitieTokenizer()
+    elif name == SpacyTokenizer.name:
+        import spacy
+        nlp = spacy.load(language, parser=False, entity=False)
```

Comment:
You are probably referring to https://github.com/golastmile/rasa_nlu/blob/pipeline/rasa_nlu/data_router.py#L19 right? Mhm till now we only used caching for the server. We could use caching for all instances though, basically always instantiating the components (like the Tokenizers) through this builder.